docs/source/tutorials/token_classification/rank.py

```diff
@@ -0,0 +1,103 @@
+import pandas as pd 
+import numpy as np 
+from cleanlab.rank import get_label_quality_scores as main_get_label_quality_scores 
+
+def get_label_quality_scores(labels: list, 
+                             pred_probs: list, 
+                             method: str = "min", 
+                             cleanlab_method: str = "self_confidence", 
+                             param: float = 0.04, 
+                             tokens: list = None, 
+                             return_token_info: bool = False): 
+    '''Returns label quality scores for each sentence 
+    
+    This is a function to compute label-quality scores for token classification datasets, where lower scores 
+    indicate labels less likely to be correct. 
+    
+    Score is between 0 and 1. 
+    
+    1 - clean label (given label is likely correct). 
+    0 - dirty label (given label is likely incorrect). 
```

Comment:
Clarify here that user should specify: `return_token_info` if they'd like to get per-token scores as well